### PR TITLE
News: Meshery Docker Extension

### PR DESCRIPTION
**Pull request by leecalcote**
**Description**

This PR announces the extension.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d3f88162c3db4fe7e45443b5725d694432d24335 at: https://627963aeef6dcc007bdd8914--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8556d93b1953e6c6c2b443643846354463a4ae8f at: https://627998f5ebab8f005f8efee8--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit fbfc9f25ad26078caf99cd43df24e0927319e5da at: https://6279e92f5ab01a33fe0b2d43--layer5ng.netlify.app

File: src/collections/blog/2019-07-09-Meshery-start-feat-WSL2-K3d/index.mdx [hunk 1]
```diff
@@ -18,7 +18,7 @@ redirect_from:
  - /blog/2019-07-09-getting-started-with-Meshery-WSL2-k3d/
 ---
 
-import { BlogWrapper } from "../Blog.style.js";
+import { BlogWrapper } from '../Blog.style.js';
```
Comment (Nikhil-Ladha) [inline src/collections/blog/2019-07-09-Meshery-start-feat-WSL2-K3d/index.mdx:21]: Considering we have the `quotes: ["error", "double"]` rule defined. I wonder how come the linter didn't throw an error on this change 🤔

Comment (leecalcote) [reply src/collections/blog/2019-07-09-Meshery-start-feat-WSL2-K3d/index.mdx:21]: Oh, double-quotes. Ok, got it.

**Comment by leecalcote** [comment]
@Nikhil-Ladha I'll close this one, understanding that we might need some graphql update in order to restructure folders.

**Pull request closed without merging**